packages/suite-desktop-core/src/modules/tray.ts

```diff
@@ -0,0 +1,146 @@
+/**
+ * Tray icon handler
+ */
+import path from 'path';
+import { Menu, Tray } from 'electron';
+
+import { DEVICE, DeviceEvent } from '@trezor/connect';
+import { Status, TraySettings } from '@trezor/suite-desktop-api/src/messages';
+import { validateIpcMessage } from '@trezor/ipc-proxy';
+
+import { app, ipcMain } from '../typed-electron';
+
+import { mainThreadEmitter, ModuleInitBackground } from './index';
+
+export const SERVICE_NAME = 'tray';
+
+export const initBackground: ModuleInitBackground = ({ store }) => {
+    const { logger } = global;
+    const initialSettings = store.getTraySettings();
+
+    const state = {
+        visible: initialSettings.showOnTray,
+        bridgeStatus: true,
+        devices: 0,
+    };
+    let tray: Tray | undefined;
+
+    const getPlatformIcon = () => {
+        switch (process.platform) {
+            case 'win32':
+                return 'trayWin.ico';
+            case 'darwin':
+                return 'trayMacTemplate.png';
+            default:
+                return 'trayLin.png';
+        }
+    };
+
+    const renderTray = () => {
+        if (!state.visible) {
+            logger.debug(SERVICE_NAME, 'Destroying tray');
+            tray?.destroy();
+            tray = undefined;
+
+            return;
+        }
+        if (!tray) {
+            logger.debug(SERVICE_NAME, 'Creating tray');
+            const iconPath = path.resolve(
+                global.resourcesPath,
+                'images/favicons/',
+                getPlatformIcon(),
+            );
+            tray = new Tray(iconPath);
+            tray.setToolTip('Trezor Suite');
+        }
+        const contextMenu = Menu.buildFromTemplate([
+            {
+                label: state.bridgeStatus
+                    ? '🟢  Trezor Bridge is running'
+                    : '🔴  Trezor Bridge not running',
+                click: () => mainThreadEmitter.emit('module/bridge/toggle'),
+            },
+            // TODO: we can add this back when connect is initialized independently of the Suite app window
+            /*{
+                label:
+                    state.devices > 0
+                        ? `🟢  Devices connected: ${state.devices}`
+                        : '🔴  No device connected',
+            },*/
+            {
+                type: 'separator',
+            },
+            {
+                label: 'Launch Trezor Suite',
+                click: () => mainThreadEmitter.emit('app/show'),
+            },
+            {
+                label: 'Stop Trezor Bridge and Quit',
```

Comment:
Again, I am not sure if we want to use the term "Trezor Bridge" at all. but this one is on @Hannsek